INSTALL.md

```diff
@@ -101,13 +101,17 @@ Create config file, based on `dendrite-config.yaml`. Call it `dendrite.yaml`. Th
 ## Starting a monolith server
 
 It is possible to use 'naffka' as an in-process replacement to Kafka when using
-the monolith server. To do this, set `use_naffka: true` in `dendrite.yaml`.
+the monolith server. To do this, set `use_naffka: true` in `dendrite.yaml`. 
+
+In the `dendrite.yaml` file, one must also uncomment the necessary line (related to naffka) to use naffka.
 
 The monolith server can be started as shown below. By default it listens for
 HTTP connections on port 8008, so point your client at
 `http://localhost:8008`. If you set `--tls-cert` and `--tls-key` as shown
 below, it will also listen for HTTPS connections on port 8448.
 
+Before running below command, ensure that all uncommented lines in the `database` section have the password of your postgres db (dendrite:password@localhost)
```

Comment:
[This line](https://github.com/matrix-org/dendrite/pull/691/files#diff-7d442b7eb49f5fc377f51e74b291cfc1R98) sort of implies this already. Did you find it was still unclear?

Reply:
Um, I thought people new to linking db to your system might need proper guidance. Should I remove it?

Reply:
Right, I was wondering if the existing instructions were confusing/undetailed and you seem to think they are. We should improve them then. I would move this advice to the Configuration section however, as having it currently only in the Monolith setup section when it also applies to the Multiprocess setup seems wrong.

Reply:
Ah sure, Ill move it along with "changes include at least" ?

Reply:
Done, recheck.